Files/ViewModels/ItemViewModel.cs

```diff
@@ -816,7 +816,9 @@ private async Task LoadItemThumbnail(ListedItem item, uint thumbnailSize = 20, I
                         var matchingStorageFolder = (StorageFolder)matchingStorageItem ?? await GetFolderFromPathAsync(item.ItemPath);
                         if (matchingStorageFolder != null)
                         {
-                            using var Thumbnail = await matchingStorageFolder.GetThumbnailAsync(ThumbnailMode.ListView, thumbnailSize, ThumbnailOptions.ReturnOnlyIfCached);
+                            var mode = thumbnailSize < 80 ? ThumbnailMode.ListView : ThumbnailMode.SingleItem;
+
+                            using var Thumbnail = await matchingStorageFolder.GetThumbnailAsync(mode, thumbnailSize, ThumbnailOptions.UseCurrentScale);
```

Comment:
Completely unrelated: could someone explain me the difference between `UseCurrentScale` and `ResizeThumbail` options? When should I use one or the other?

Reply:
My understanding is that resize thumbnail resizes the thumbnail (up or down) to match the requested size, while increase size just scales up the size to match the scaling of the display.

Reply:
So `UseCurrentScale` should be used all the times?

Reply:
Yeah I don't think it would make much of a difference.

Reply:
Thanks. I believe we do the opposite in most other places xD